doc/release/multilocale.rst

```diff
@@ -4,233 +4,198 @@
-           export GASNET_CSPAWN_CMD="srun -N%N %C"
-
-
-      #. To simulate multiple Chapel locales with one workstation,
-         you can request that GASNet spawn its child functions on
-         your local machine using:
-
-         .. code-block:: bash
-
-           export GASNET_SPAWNFN=L
+   based on these settings.
 
-         If you have problems with the local spawn, you may need to
-         set ``GASNET_MASTERIP`` to your IP address.
-
-
-      #. For other options, please refer to:
-
-         ``$CHPL_HOME/third-party/gasnet/gasnet-src/udp-conduit/README``
-
-   #. If you are using GASNet's ibv conduit, configure the environment to
-      launch program instances.  We have experience with these configurations:
-
-      i. For clusters using SLURM, enable slurm-gasnetrun_ibv:
-
-         .. code-block:: bash
-
-           export CHPL_LAUNCHER=slurm-gasnetrun_ibv
-
-         See :ref:`readme-launcher` for other options available, such
-         as setting the time limit or selecting the type of node.
-
-
-      #. To launch InfiniBand jobs with SSH, use the following
-
-         .. code-block:: bash
-
-           export CHPL_LAUNCHER=gasnetrun_ibv
-           export GASNET_SSH_SERVERS="host1 host2 host3 ..."
-           export GASNET_IBV_SPAWNER=ssh
-
-
-      #. We've observed job launch hangs in some systems with InfiniBand
-         that were resolved by limiting the memory available for
-         communication, for example with:
-
-         .. code-block:: bash
-
-           export GASNET_PHYSMEM_MAX=1G
-
-
-      #. For more information on these and other available GASNet options
-         other options, including configuring to launch through MPI,
-         please refer to:
-
-         ``$CHPL_HOME/third-party/gasnet/gasnet-src/ibv-conduit/README``
+#. Compile your Chapel program as usual:
 
-   #. If you are trying to debug job launch, try adding ``-v`` to your
-      program's command line and set:
+   .. code-block:: bash
 
-      .. code-block:: bash
+     chpl -o hello6-taskpar-dist $CHPL_HOME/examples/hello6-taskpar-dist.chpl
 
-        export GASNET_VERBOSEENV=1
+#. Set any environment variables necessary to control the launcher.
+   See :ref:`readme-launcher` or documentation for your platform.
+   For example, to specify which servers the UDP conduit should use as
+   described in :ref:`using-udp`:
 
-   #. We've had best results with console I/O using:
+   .. code-block:: bash
 
-      .. code-block:: bash
+     export GASNET_SPAWNFN=S
+     export GASNET_SSH_SERVERS="host1 host2 host3 ..."
 
-        export GASNET_ROUTE_OUTPUT=0
+#. Specify the number of locales on the command line. For example:
 
-      but your mileage may vary.
+   .. code-block:: bash
 
+     ./hello6-taskpar-dist -nl 2
 
-#. Compile your Chapel program as usual:
+   runs our Hello World program on 2 locales.
+
+What is this _real program?
++++++++++++++++++++++++++++
+
+When you compile a Chapel program for multiple locales, you should
```

Comment:
Yes, if you set CHPL_LAUNCHER=none but CHPL_COMM=something you can have a multilocale build without two binaries. But that is not a typical configuration.